independent-projects/tools/registry-client/src/main/java/io/quarkus/registry/catalog/json/JsonCatalogMapperHelper.java

```diff
@@ -0,0 +1,64 @@
+package io.quarkus.registry.catalog.json;
+
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.PropertyNamingStrategies;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import io.quarkus.maven.ArtifactCoords;
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.Reader;
+import java.io.Writer;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+public class JsonCatalogMapperHelper {
+
+    private static ObjectMapper mapper;
```

Comment:
Can this mapper be set externally somehow?

Reply:
I added methods to pass in a custom mapper. Would that work?

Reply:
Yes, I think that would help. I was mostly thinking of reusing the `ObjectMapper` produced by the `quarkus-jackson` extension